src/components/Thumbnail/Thumbnail.e2e.ts

```diff
@@ -0,0 +1,17 @@
+import * as eyes from 'eyes.it';
+import {browser} from 'protractor';
+import {getStoryUrl, waitForVisibilityOf} from 'wix-ui-test-utils/protractor';
+import {thumbnailTestkitFactory} from '../../testkit/protractor';
+
+describe('Thumbnail', () => {
```

Comment:
Missing screenshots for :
- title + image
- title + description
- title + image + description

Reply:
A lot of components (maybe all of them) are lacking many screenshots, mainly because the exampleProps is not yet working or we don't even know what exactly we want to do in e2e's.

I really want us to get a decision about this, but for now, I don't see why to treat this component any differently